### ISSUE TITLE
Formatting breaks code

### ISSUE THREAD

**Issue by schjonhaug** [nicklockwood/SwiftFormat#627]
Formatting

```
var activeURLs = Dictionary<URL, Array<((_ image: Data?)->Void)>>()
```

with standard rules gives

```
var activeURLs = [URL: [(_ image: Data?) -> Void]]()
```

which does not compile.

**Comment by nicklockwood** [comment]
@schjonhaug thanks for reporting, I'll try to fix this. In the meantime you can work around it either by adding

```bash
--disable typeSugar
```

To your SwiftFormat CLI arguments, or by adding the following comment above that line in your code:

```swift
// swiftformat:disable:next typeSugar
var activeURLs = [URL: [(_ image: Data?) -> Void]]()
```

**Comment by nicklockwood** [comment]
Fixed in 0.44.11
